multiqc/modules/mapdamage/mapdamage.py

```diff
@@ -0,0 +1,282 @@
+        self.lgdist_fw_data = dict()
+        self.lgdist_rv_data = dict()
+
+        # Find and load log files
+        for f in self.find_log_files("mapdamage", filehandles=True):
+            self.parseInput(f)
+
+        # Filter to strip out ignored sample names
+        self.threepGtoAfreq_data = self.ignore_samples(self.threepGtoAfreq_data)
+        self.fivepCtoTfreq_data = self.ignore_samples(self.fivepCtoTfreq_data)
+        self.lgdist_fw_data = self.ignore_samples(self.lgdist_fw_data)
+        self.lgdist_rv_data = self.ignore_samples(self.lgdist_rv_data)
+
+        ## No need to run self.write_data_file as all the data imported is already in a TSV format and can be (almost) directly used for plotting.
+
+        # Basic Stats Table, use generic function to add data to general table
+        self.dmgprof_misinc_stats(self.threepGtoAfreq_data, "3 Prime", "G>A")
+        self.dmgprof_misinc_stats(self.fivepCtoTfreq_data, "5 Prime", "C>T")
+
+        # Add plots
+        if len(self.threepGtoAfreq_data) > 0:
+            self.add_section(
+                name="3P misincorporation plot",
+                description="3' misincorporation plot for G>A substitutions",
+                helptext="""
+                This plot shows the frequency of G>A substitutions at the 3' read ends. Typically, one would observe high substitution percentages for ancient DNA, whereas modern DNA does not show these in higher extents.
+                """,
+                plot=self.threeprime_plot(),
+            )
+        if len(self.fivepCtoTfreq_data) > 0:
+            self.add_section(
+                name="5P misincorporation plot",
+                description="5' misincorporation plot for C>T substitutions",
+                helptext="""
+                This plot shows the frequency of C>T substitutions at the 5' read ends. Typically, one would observe high substitution percentages for ancient DNA, whereas modern DNA does not show these in higher extents.
+                """,
+                plot=self.fiveprime_plot(),
+            )
+        if len(self.lgdist_fw_data) > 0 and len(self.lgdist_rv_data) > 0:
+            self.add_section(
+                name="Forward read length distribution",
+                description="Read length distribution for forward strand (+) reads.",
+                helptext="""
+                This plot shows the read length distribution of the forward reads in the investigated sample. Reads below lengths of 30bp are typically filtered, so the plot doesn't show these in many cases. A shifted distribution of read lengths towards smaller read lengths (e.g around 30-50bp) is also an indicator of ancient DNA.
+                """,
+                plot=self.lgdistplot(self.lgdist_fw_data, "Forward"),
+            )
+            self.add_section(
+                name="Reverse read length distribution",
+                description="Read length distribution for reverse strand (-) reads.",
```

Comment:
Minor detail - using a minus sign Unicode character instead of a dash (so it's the same size as the plus sign and looks neater):
```suggestion
                description="Read length distribution for reverse strand (−) reads.",
```